datadog/README.md

```diff
@@ -0,0 +1,15 @@
+# Datadog
+
+Add a button to the Tilt UI to toggle Datadog agent resources using Helm.
+
+## Usage
+
+After registering the repo and extension (see [main README](../README.md)), you can invoke the extension using 
+`datadog_up` in your Tiltfile. This will add a button to the global Tilt UI (a paw print icon) which can toggle 
```

Comment:
Trailing whitespace

```suggestion
`datadog_up` in your Tiltfile. This will add a button to the global Tilt UI (a paw print icon) which can toggle
```